KLR/Core/Indexing.lean

```diff
@@ -206,7 +206,7 @@ theorem AccessBasic.getShape_length (a : AccessBasic) :
   · rename_i heq; apply (AccessBasic.shape.noFail _ heq).elim
 
 @[simp] def Shape.freeDim (s : Shape) : Nat := s.freeDims.length
-@[simp] def TensorName.freeDim (s : TensorName) : Nat := s.shape.freeDim
+@[simp] def TensorSRam.freeDim (s : TensorSRam) : Nat := s.shape.freeDim
```

Comment:
nit: SRam is a weird capitalization. I'd get Sram and SRAM. In one repo I worked in (forgot, was a while ago) there was a convention that no matter whether a word was an acronym or not, it was always CamelCased. I like that convention, but not wedded to it.